guestbook.module

```diff
@@ -0,0 +1,177 @@
+<?php
+module_load_include('inc', 'guestbook', 'include/guestbook_db');
+module_load_include('inc', 'guestbook', 'include/guestbook_core');
+module_load_include('inc', 'guestbook', 'include/guestbook.theme');
+module_load_include('inc', 'guestbook', 'include/guestbook_weather');
+
+/**
+* @file
+* Build page, form for guestbook and weather block.
+*/
+
+/**
+* Implements hook_menu(). 
+*/
+ function guestbook_menu() {
+   $items = array();
+   // Main page.
+   $items['guestbook-page'] = array(
+     'title' => 'Guestbook',
+     'page callback'   => 'drupal_get_form',
+     'page arguments'  => array('guestbook_page_form'),
+     'access callback' => TRUE,
+  );
+   // Edit message.
+   $items['guestbook-page/%mid/edit'] = array(
+     'title'            => 'Guestbook',
+     'page callback'    => 'drupal_get_form',
+     'page arguments'   => array('guestbook_page_form', 1),
+     'access callback'  => TRUE,
+     'type'             => MENU_CALLBACK,
+
+   );
+   // Delete message.
+   $items['guestbook-page/%mid/delete'] = array(
+     'title'            => 'Delete message',
+     'page callback'    => 'guestbook_message_delete',
+     'page arguments'   => array(1),
+     'access callback'  => TRUE,
+     'type'             => MENU_CALLBACK,
+   );
+   return $items;
+}
+
+/**
+ * Implements hook_form.
```

Comment:
this is not a hook, it's just a callback. Move this function to guestbook.form.inc